### PR TITLE
Add SAving time static site post

### PR DESCRIPTION
**Pull request by tucq88**

### CODE REVIEW

**Comment by tucq88** [comment]
@hgb123 I can't preview this page on local, it not displayed with `hugo serve -D`, also not be generated with `hugo -D` , please help.

**Comment by hgb123** [comment]
@tucq88 The problem seems to lie in timezone. I made a commit, please have a look.

File: content/post/saving-time-with-static-site.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Saving time with static sites
+author: TuCQ
+date: 2019-12-02 09:30:00+07:00
+---
```
Comment (hgb123) [inline content/post/saving-time-with-static-site.md:6]: Could you please add summary for this post?

**Comment by tucq88** [comment]
Done, actually Python post also doesn't have summary part.

**Comment by hgb123** [comment]
@tucq88 We have to urge (or force) the author of that post to do that..

**Comment by tucq88** [comment]
Could you please update the README on how to do so?

**Comment by hgb123** [comment]
@tucq88 noted, I will update soon

**Pull request merged**